ext_mongodb.php

```diff
@@ -667,7 +667,31 @@ function bsonUnserialize(array $data) : void;
 
 final class Binary implements Type, \Serializable
 {
-	use DenySerialization;
+	static private function checkArray(array $state)
+	{
+		if (
```

Comment:
Were you always using tabs for indentation in the PHP source? I don't recall noticing that before.